prisma/dev-seeds/user.ts

```diff
@@ -201,6 +201,17 @@ const users: User[] = [
         createdAt: new Date(),
         updatedAt: null,
     },
+    {
+        id: 10018,
+        firstName: "Greg",
+        lastName: "Maxin",
+        email: "gmaxin+admin@uwaterloo.ca",
+        emailVerified: new Date(),
+        role: roles.PROGRAM_ADMIN,
+        image: null,
+        createdAt: new Date(),
+        updatedAt: null,
+    },
```

Comment:
@GodGreg please add a ParentAdmin record in the prisma/dev-seeds/program-admin.ts file.